### PR TITLE
Added new get environment functionality

### PR DESCRIPTION
**Pull request by luigizuccarelli**

### CODE REVIEW

File: commands/get/environments.go [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+	newrequest, err := http.NewRequest("GET", fullURL, nil)
+	if err != nil {
+		return cli.NewExitError("could not create new request object "+err.Error(), 1)
+	}
+
+	// Use the fh-mbass-api-key
+	// I have hardcoded this for now - not ideal
+	// Looking into doing a query to get this key
+
+	newrequest.Header.Add("X-FH-AUTH-USER", "9052dfadcf8994d997e6d1fa7c395d760e448312")
```
Comment (maleck13) [inline commands/get/environments.go]: yeah maybe we could take an apiKey as part of the login command. Also kinda suprised this doesn't work with the cookie. What was the error you were getting back?

**Comment by maleck13** [comment]
@luigizuccarelli this is looking great. I can really see you're starting to get the hang of the code 👍

**Comment by luigizuccarelli** [comment]
@maleck13 - Thanks for the review. I will look into getting the api/s on the weekend. Must be honest I'm really enjoying this. 

Thanks again for getting us involved :+1:

File: commands/get/environments.go (hunk 1 above)
Comment (luigizuccarelli) [inline commands/get/environments.go]: This is the response I got using userData.Auth
```
{"userDetail":"Permission denied!","systemDetail":"Permission denied! - 'Permission denied!'","code":"FH-SUPERCORE-ERROR"}
 response 403 
```

**Pull request merged**